docs/source/supergraphs.md

```diff
@@ -74,3 +74,15 @@ rover supergraph compose --config ./supergraph.yaml > prod-schema.graphql
 ```
 
 > For more on passing values via `stdout`, see [Using `stdout`](./conventions#using-stdout).
+
+#### Gateway compatibility
+
+`rover supergraph compose` utilizes the composition function published in the [harmonizer](https://crates.io/crates/harmonizer) package under the hood. Harmonizer is still under rapid development, and some breaking changes are to be expected. Newer versions of Rover emit supergraph schemas that are incompatible with older versions of [`@apollo/gateway`](https://www.npmjs.com/package/@apollo/gateway).
+
+The following table should help you understand the compatible versions of these two pieces of software.
+
```

Comment:
I might suggest keeping harmonizer — even as a word – as an implementation detail.  "Harmonizer" was originally just meant to be a _name of a package_ that we could publish to Crates.io that was roughly synonymous with "composition" (and _available_).

I'll let @StephenBarlow make suggestions, too, but perhaps something along the lines of:

```suggestion
The `rover supergraph compose` command produces a supergraph by utilizing the composition functions of the [`@apollo/federation`](https://www.apollographql.com/docs/federation/api/apollo-federation/) package.  As that library is still in pre-1.0 releases (as are Rover and Apollo Gateway), some updates to Rover may result in a supergraph with new functionality and therefore necessitate complimentary updates to Apollo Gateway.

Apollo Gateway will fail to start-up if it's provided with a supergraph that it doesn't support, so it's recommended to test launching Apollo Gateway with the supergraph it will be ultimately be deployed with in a CI pipeline to ensure compatible versions.

We aim to reduce the frequency at which these paired updates are necessary by making supergraph additions backwards-compatible.  We will note changes which require corresponding Apollo Gateway updates  clearly in the Rover _Release Notes_ as well as updating the compatibility table below.
```

Reply:
Ok, I think at the very least I may actually necessitate the grace that @StephenBarlow could bring to what I wrote, but I think the spirit of what I wrote is accurate. 😆